src/lucky/tags/custom_tags.cr

```diff
@@ -32,4 +32,10 @@ module Lucky::CustomTags
     yield
     @view << "</#{name}>"
   end
+
+  def inline_tag(name : String, options = EMPTY_HTML_ATTRS, **other_options)
```

Comment:
Oh and could you add some brief docs for this new method noting that it will not be generated with an end tag?